FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemoTests/FBSnapshotTestCaseDeviceAgnosticTests.m

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Copyright (c) 2013, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import <FBSnapshotTestCase/FBSnapshotTestCase.h>
+
+@interface FBSnapshotTestCaseDeviceAgnosticTests : FBSnapshotTestCase
+
+@end
+
+@implementation FBSnapshotTestCaseDeviceAgnosticTests
+
+- (void)setUp
+{
+    [super setUp];
+    self.deviceAgnostic = YES;
+    self.recordMode = NO;
+}
+
+- (void)testSuccessfulComparisonOfDeviceAgnosticReferenceImages
```

Comment:
Interestingly, it's already run on both 5 & 6 iPhone models, but the application itself isn't filling the screen:

![image](https://cloud.githubusercontent.com/assets/444217/12079129/e09a2660-b1fb-11e5-9091-cb2ff438d7ca.png)

I'll see if I can fix this quickly.  Good catch that this should be verified for different screen sizes.